README.md

```diff
@@ -1,7 +1,7 @@
-# Black Magic Probe
-
 [![Discord](https://img.shields.io/discord/613131135903596547?logo=discord)](https://discord.gg/P7FYThy)
 
+# Black Magic Probe
```

Comment:
Ah, I was not aware of that rule, I moved the badge to the top because I think it looks better there, it looked to me like it was breaking the flow between the project name and the description

We can move it to additional resources? though that would make it *slightly* harder to spot

Reply:
It does, but we see what you mean about flow - think this needs further reworking with build badges and the like, but that's for another change and time - not this round.

It looks fine in the resources section, so going with that for now.